### ISSUE TITLE
Problem.

### ISSUE THREAD

**Issue by iPetrov2** [finndev/PokeBuddy#1003]
Im using VPN for Pokebuddy and when i start pokebuddy it goes to pokestops but i get not xp from them

**Comment by ChiisanaKoneko** [comment]
@iPetrov2  that sounds like a soft ban

**Comment by AlexCtasy** [comment]
Dont use anymore pokebuddy man, is like that we all getting ban just cause using the pokebuddy..

**Comment by ChiisanaKoneko** [comment]
@AlexCtasy its you guys that are doing something ive been using this and never received a ban yet so far and its been weeks

**Comment by AlexCtasy** [comment]
@ChiisanaKoneko Yes ok! happy that u never gets banned, but I have same error like this guy and happened when i started use this pokebuddy. Since I used yesterday and today i cannot take exp from pokestops

**Comment by iPetrov2** [comment]
@ChiisanaKoneko i thought using VPN you wouldnt Get SoftBanned. fml

**Comment by AlexCtasy** [comment]
U thought xoaxoa.

**Comment by AlexCtasy** [comment]
U get btw.

**Comment by iPetrov2** [comment]
@AlexCtasy The Pokemon Servers in The U.S Are Slow
lol. im not using my main PoGo Account so im not that worried. Kinda

**Comment by ChiisanaKoneko** [comment]
K think like this different ips will be a sure sign of a bot/etc

**Comment by ChiisanaKoneko** [comment]
my phones ip is in the same group as my laptop therefore looks normal
